### ISSUE TITLE
v1.10 - error message in the logs about the graphQL addon

### ISSUE THREAD

**Issue by bcollard** [solo-io/gloo#5793]
### Gloo Edge Version

1.10.x (latest stable)

### Kubernetes Version

1.21.x

### Describe the bug

Some users have observed the following error message in the logs of the Gloo pod, after an upgrade from 1.9 to 1.10:

```
gloo edge license is not valid or does not have graphql addon, not translating graphql configuration
```

Generating a new license key solved the issue

### Steps to reproduce the bug

I was not able to reproduce with the license they were using.

### Expected Behavior

no WARNING message in the logs

**Comment by rinormaloku** [comment]
This seems quite a good log. maybe just switch the logging level to info? as it is for informational purposes

**Comment by pmichna** [comment]
I'm confirming the problem. We don't need the GraphQL add-on and thus the warnings just litter our logs (13 400 log entries per hour).

Could you reopen the issue?

**Comment by sam-heilbron** [comment]
This has been resolved in 1.10 and 1.11. If a license is not enabled for graphql, we will not process the graphql features, as well as log to the debug level.

Additionally, as part of the work to support https://github.com/solo-io/gloo/issues/5651, if a filter is not even configured in the Gloo API, we will not process the graphql translation.
